scripts/generate-markdown/markdown.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const fs = require('fs');
+const markdownMagic = require('markdown-magic'); // eslint-disable-line import/no-extraneous-dependencies
+
+const defaults = {
+  dir: './packages',
+  verbose: false,
+  bullet: '*',
+};
+
+const header = `| Terra Package      | Version | Status | Dependencies |
+|--------------------|---------|--------|--------------|
+`;
+
+const config = {
+  transforms: {
+    // see: https://github.com/camacho/markdown-magic-subpackage-list
+    SUBPACKAGELIST(content, _options, configuration) {
+      const options = Object.assign({}, defaults, _options);
+
+      const packagesDir = path.resolve(
+        path.dirname(configuration.originalPath), options.dir,
+      );
+
+      return header + fs
+        .readdirSync(packagesDir)
+        .map(filename => path.join(packagesDir, filename))
+        .filter(filePath => fs.statSync(filePath).isDirectory())
+        .filter(dirPath => fs.existsSync(path.join(dirPath, 'package.json')))
+        .map(dirPath => [
+          path.relative(path.dirname(configuration.originalPath), dirPath),
+          JSON.parse(fs.readFileSync(path.resolve(dirPath, 'package.json'))),
```

Comment:
The list of directories returned by `fs.readdirSync` will map to their package.json's name. Do we need to load the package.json, or could we filter on
```
.filter(dirPath => fs.existsSync(path.join(dirPath, 'package.json')))
```
and then map the directory name?

Reply:
It looks like, for what this is trying to accomplish in this repo, the only thing we really need to grab is the package name, and hard-code the paths. I drastically shortened the steps it goes through, let me know your thoughts!

Reply:
It def looks a bit cleaner to understand!